R/ttest_Paired.qmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "Paired t-test"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# **Paired t-test in R**
+
+The Paired t-test is used when two samples are naturally correlated. In the Paired t-test, the difference of the means between the two samples is compared to a given number that represents the null hypothesis. For a Paired t-test, the number of observations in each sample must be equal.
+
+In R, a Paired t-test can be performed using the Base R `t.test()` from the **stats** package or the `proc_ttest()` function from the **procs** package.
+
+## Normal Data {#normal}
+
+By default, the R paired t-test functions assume normality in the data and use a classic Student's t-test.
+
+### Data Used
+
+The following data was used in this example.
+
+```{r eval=FALSE, echo = TRUE}
```

Comment:
Please make your code runnable where possible, that was we know everything is still working

Reply:
It should be runnable.  Is it not running, or are you talking about the "eval=FALSE"?